ethereum/core/src/main/java/tech/pegasys/artemis/core/EpochProcessorUtil.java

```diff
@@ -612,22 +610,22 @@ public static void process_final_updates(MutableBeaconState state) {
     for (int index = 0; index < validators.size(); index++) {
       Validator validator = validators.get(index);
       UnsignedLong balance = balances.get(index);
-      long HALF_INCREMENT = Constants.EFFECTIVE_BALANCE_INCREMENT / 2;
-      if (balance.compareTo(validator.getEffective_balance()) < 0
-          || validator
-                  .getEffective_balance()
-                  .plus(UnsignedLong.valueOf(3 * HALF_INCREMENT))
-                  .compareTo(balance)
-              < 0) {
 
+      final UnsignedLong hysteresis_increment =
+          EFFECTIVE_BALANCE_INCREMENT.dividedBy(HYSTERESIS_QUOTIENT);
```

Comment:
Seems like these 3 variables could go into the Constants file

Reply:
If the constants were actually constants yes, but we'd need to be able to reset them after loading from the constants yaml file.